graviti/openapi/request.py

```diff
@@ -9,7 +9,7 @@
 from uuid import uuid4
```

Comment:
It looks like the file should be uniformly named "requests.py"?

Reply:
"utility/request" -> "client/requests"? @Lee-000

Reply:
We could do that in another pr.

Reply:
> We could do that in another pr.

ok